components/hab/src/command/studio/enter.rs

```diff
@@ -17,6 +17,7 @@ use crate::{config,
 use habitat_core::AUTH_TOKEN_ENVVAR;
 
 pub const ARTIFACT_PATH_ENVVAR: &str = "ARTIFACT_PATH";
+pub const SSL_PATH_ENVVAR: &str = "SSL_PATH";
```

Comment:
Should this be prefixed with `HAB_`?

Reply:
Ah, this needs to be "CERT_PATH" to match what the studio is actually expecting

Reply:
Fixed